hivemind_core/protocol.py

```
@@ -705,34 +866,308 @@ def handle_escalate_message(
+        """
+        bus = self.get_bus(client)
+
+        # Set up a one-shot listener for the response
+        response_holder: List[Message] = []
+        injected_type = bus_message.msg_type
+
+        def _on_response(msg: Union[Message, str]):
+            if isinstance(msg, str):
+                try:
+                    msg = Message.deserialize(msg)
+                except Exception:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)